json/src/main/java/com/basistech/rosette/dm/jackson/KnownKey.java

```diff
@@ -0,0 +1,69 @@
+/*
+* Copyright 2018 Basis Technology Corp.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package com.basistech.rosette.dm.jackson;
+
+import com.basistech.rosette.dm.UnknownAttribute;
+import com.basistech.util.LanguageCode;
+
+/**
+ * The attributes currently defined in the data model.
+ * Applications may extend the model with additional attributes;
+ * this provides keys for {@link com.basistech.rosette.dm.AnnotatedText}
+ * for the known map keys.
+ */
+public enum KnownKey {
+    // TODO: Can we remove this enum somehow?
+    STRING("string", String.class),
```

Comment:
Do you see these be lower cased in json? If so, we might as well break the coding convention here so that we don't need to write custom jackson ser/deserializers.

Reply:
Can you elaborate on this? I'm not sure what you're getting at.